content/hi/cloud_native_security.md

```diff
@@ -0,0 +1,27 @@
+--- 
+
+Title: क्लाउड नेटिव सुरक्षा 
+
+Status: completed 
+
+Category: अवधारणा 
+
+--- 

```

Comment:
```suggestion
--- 
Title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
Status: completed 
Category: अवधारणा 
--- 

```

Reply:
Made changes as per you suggested

Thank you @anubha-v-ardhan :+1: